mythril/laser/plugin/plugins/plugin_annotations.py

```diff
@@ -42,18 +48,31 @@ def __copy__(self):
 
     def get_storage_write_cache(self, iteration: int):
         if iteration not in self.storage_written:
-            self.storage_written[iteration] = []
+            self.storage_written[iteration] = set()
 
         return self.storage_written[iteration]
 
     def extend_storage_write_cache(self, iteration: int, value: object):
         if iteration not in self.storage_written:
```

Comment:
The same would probably work here

```suggestion
        self.storage_written.get(iteration, set()).add(value)
```